app/components/UI/Navbar/index.js

```diff
@@ -716,17 +760,16 @@ export function getTransparentOnboardingNavbarOptions(themeColors) {
  *
  * @returns {Object} - Corresponding navbar options containing headerTitle and a back button
  */
-export function getTransparentBackOnboardingNavbarOptions(themeColors) {
+export function getTransparentBackOnboardingNavbarOptions(themeColors, color) {
```

Comment:
This seems to be only used at one place, and never with the second argument
https://github.com/MetaMask/metamask-mobile/blob/351a27625d9ed4f6ffa55ca063ddb3bceb0e1e74/app/components/Views/Onboarding/index.js#L228

If I'm not missing something and if it's that the case could we clean it and have it with just one param?
Also it seems that `color` is not an appropriate name, would be more accurate to name it `backgroundColor`, wdyt?